appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/rest/init/OpenApiServletContainerInitializer.java

```diff
@@ -91,6 +91,7 @@ public void onStartup(Set<Class<?>> c, ServletContext ctx) throws ServletExcepti
 
```

Comment:
I believe the initializer is not doing anything now, since you're instantiating the servlet container in lines below. Could you try if it works without it?

Reply:
Thanks @pdudits , Also as of now @ApplicationPath annotation is removed from `fish.payara.microprofile.openapi.rest.app.OpenApiApplication` so the initializer call is redundant now.